### PR TITLE
Operator for review

### PR DESCRIPTION
**Pull request by adamlocker**
Operator entity for review. It's a bit disappointing...

### CODE REVIEW

**Comment by JDSFSA** [comment]
could it be simplified to something along the lines of "An operator can be defined as a person or company that runs a business"?

**Comment by Jamie-Atkinson** [comment]
I would go the other way JD, the definition i was using that the operator is the natural or legal person/persons responsible for ensuring that the requirements of food law are met within the food buisness under their control. The operator is not always the person that runs the buisness i would expect. opinions @adamlocker ?

File: models/drafts/operator.md [hunk 1]
```diff
@@ -32,16 +32,18 @@ As the Operator can be a legal entity, such as a limited company, it is not appr
 -   Trader
 
 ### What it is not
-The operator in this context is the entity which is associated with the `Establishment` which is being regulated under a scheme. Frequently operators can be owned by other entities. These entities are not the operator.
+The operator in this context is the entity which is associated with the `Establishment` which is being regulated under a scheme. Frequently operators can be owned by other legal entities. These entities are not the operator.
```
Comment (Jamie-Atkinson) [inline models/drafts/operator.md]: Do we want to reference that there are other types of operator that, under our definition would not be counted and thus should not use the operator entity i.e. operator of heavy machinery?

File: models/drafts/operator.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ Experimental
...
+Operator is the person or legal entity responsible for running a food business. When considered in tandem with `Premises`, the combined entity is an `Establishment`.
 
 The operator can be a `physical person` or a `legal person` and we sometimes use these terms to distinguish between the two. In data terms, these entities should be considered children of Operator, as they have different contextual properties.
 
-As the Operator can be a legal entity, such as a limited company, it is not appropriate for example to use the Operator entity if you were building an inspection service or similar. It can sometimes be necessary to identify a separate `Contact` entity.
+As the Operator can be a legal entity, such as a limited company, it is not necessarily appropriate to use the Operator entity if you were building an inspection service or similar. It can sometimes be necessary to identify a separate `Contact` entity.
 
 ### Related entities
 `Physical person`, `Legal person` and `Contact` are considered child entities of person.
```
Comment (Jamie-Atkinson) [inline models/drafts/operator.md]: Why is contact a child. they are related agreed but is it a child?

File: models/drafts/operator.md (hunk 1 above)
Comment (JDSFSA) [reply models/drafts/operator.md]: > I would go the other way JD, the definition i was using that the operator is the natural or legal person/persons responsible for ensuring that the requirements of food law are met within the food buisness under their control. The operator is not always the person that runs the buisness i would expect. opinions @adamlocker ?

I see where your coming from as there is generally a person as a contact but the "operator" can be a company, the company would then appoint a person to act as a contact but that person isn't the operator as the operator can be a company.

Comment (pc42) [reply models/drafts/operator.md]: found the following doc - bottom of page 17 has the same definition as Jamie https://www.food.gov.uk/sites/default/files/media/document/fsa1782002guidance.pdf

I'm happy to go with the above definition.

on the FSA register a food business site it states 'Companies involved with food distribution or food supply that operate from an office should also register as food businesses. This applies even if no food is kept at the premises.'  Would be interesting to know exactly what data is collected if it is a company - might we need two different sets of Key data depending on whether a person(s) or company.  Or do we always collect details of a named individual who is nominated to act on behalf of said company

Unless we make references to machine operators (eg in plants) I don't think we need to include in the what it is not section

Comment (Jamie-Atkinson) [reply models/drafts/operator.md]: @JDSFSA legal person is referencing a company, natural is referencing well a real person, I saw this language a few times online, but given you did not imediatly grasp it i think it should be improved.

File: models/drafts/operator.md (hunk 2 above)
Comment (adamlocker) [reply models/drafts/operator.md]: You are right, it's actually a related entity but not a child. Maybe a sibling but could just have a weak relationship. https://docs.google.com/presentation/d/1T1aqqhagB_1jWZK7UA-tUDtPZWoIa6EbPypL7uyuQxY/edit#slide=id.g75cd32ba58_0_468

File: models/drafts/operator.md (hunk 1 above)
Comment (adamlocker) [reply models/drafts/operator.md]: Shoud answer on the other operators is no; I don't think it help implement a data model when building services.

Comment (adamlocker) [reply models/drafts/operator.md]: I think the named person thing is covered by `Contact`

Comment (adamlocker) [reply models/drafts/operator.md]: Rewrite to make physical person and legal person propeties contectual to simplify modelling.

**Pull request merged**